services/web-app/pages/assets/index.js

```diff
@@ -193,6 +199,167 @@ const PageContent = () => {
           </Grid>
         ))}
       </div>
+      <Grid className={styles.hightlightGrid}>
+        <Column sm={4} md={8} lg={8} xlg={7}>
+          <p className={clsx(styles.subheadingMedium)}>
+            The <span className={styles.bold}>asset discovery experience</span> is your efficient
```

Comment:
I'm not super familiar with best practices for textual stuff in HTML, but would a `<strong>` element work here as opposed to a span+class combo?

Reply:
Ooo good point - I might still need to add a class - but according to this we should be using `<strong>`

https://www.geeksforgeeks.org/difference-between-strong-and-bold-tag-in-html/#:~:text=The%20main%20difference%20between%20these,text%20conventionally%20styled%20in%20bold.